### DISCUSSION TITLE
Retrieve Items of Collection

### DISCUSSION THREAD

**Discussion by nithin-techforce** [directus/directus#5623] [Archive]
hi , I want to retrieve the items of a collection based on specific column criteria for example: where column_name="value" and Column_name="value" where in documentation I see retrieving items is based only 1 criteria which is autogenerated id where we cant track the data whith that id

thank you

**Comment by rijkvanzanten** [comment]
Did you see the `filter` query param? https://docs.directus.io/reference/api/query/#filter
